### ISSUE TITLE
unable to install "typings" folder via npm

### ISSUE THREAD

**Issue by uniqueone2000** [npm/npm#13320]
#### I'm opening this issue because:

  - [ ] npm is crashing.
  - [ ] npm is producing an incorrect install.
  - [ ] npm is doing something I don't understand.
  - [x] Other (_see below for feature requests_):

#### How can the CLI team reproduce the problem?
I received the error in the terminal when attempted to run the following command: 
sudo npm run typings install

### supporting information:

 - `npm -v` prints:
 - `node -v` prints:
 - `npm config get registry` prints:
 - Windows, OS X, or Linux?:
 - Network issues:
   - Geographic location where npm was run:
   - [ ] I use a proxy to connect to the npm registry.
   - [ ] I use a proxy to connect to the web.
   - [ ] I use a proxy when downloading Git repos.
   - [ ] I access the npm registry via a VPN
   - [ ] I don't use a proxy, but have limited or unreliable internet access.
 - Container:
   - [ ] I develop using Vagrant on Windows.
   - [ ] I develop using Vagrant on OS X or Linux.
   - [ ] I develop / deploy using Docker.
   - [ ] I deploy to a PaaS (Triton, Heroku).

#### What's the feature?
I receive error messages when attempting to install the "typings" folder via nam as discussed in the Angular2 tutorial at this link: 
https://angular.io/docs/ts/latest/quickstart.html#!#create-and-configure

#### What problem is the feature intended to solve?
This quote is directly from the Angular website and should explain this question: 
QuickStart itself doesn't require these typings but many of the documentation samples do. Most of us would be disappointed if we couldn't code against typical ES2015 features or support testing right out-of-the-box.

#### Is the absence of this feature blocking you or your team? If so, how?
I have not progressed further with this tutorial so I'm not sure

#### Is this feature similar to an existing feature in another tool?
???

#### Is this a feature you're prepared to implement, with support from the npm CLI team?

Here is the link to my nom-debug.log file: https://gist.github.com/uniqueone2000/93b84f3ef7b3a63cee072c1d0d24e1a4
-->

**Comment by KenanY** [comment]
@uniqueone2000 Can I see your `package.json`?

**Comment by uniqueone2000** [comment]
Sorry I missed you an hour ago. Here it is:

{
  "dependencies": {
    "@angular/common": "2.0.0-rc.4",
    "@angular/compiler": "2.0.0-rc.4",
    "@angular/core": "2.0.0-rc.4",
    "@angular/forms": "0.2.0",
    "@angular/http": "2.0.0-rc.4",
    "@angular/platform-browser": "2.0.0-rc.4",
    "@angular/platform-browser-dynamic": "2.0.0-rc.4",
    "@angular/router": "3.0.0-beta.1",
    "@angular/router-deprecated": "2.0.0-rc.2",
    "@angular/upgrade": "2.0.0-rc.4",
    "systemjs": "0.19.27",
    "core-js": "^2.4.0",
    "reflect-metadata": "^0.1.3",
    "rxjs": "5.0.0-beta.6",
    "zone.js": "^0.6.12",
    "angular2-in-memory-web-api": "0.0.14",
    "bootstrap": "^3.3.6"
  },
  "devDependencies": {
    "concurrently": "^2.0.0",
    "lite-server": "^2.2.0",
    "typescript": "^1.8.10",
    "typings": "^1.0.4"
  }
}

**Comment by KenanY** [comment]
How come the one I see on the site you linked to looks different?

![2016-07-08-09-17-37-0587459](https://cloud.githubusercontent.com/assets/733364/16688659/02a34618-44ed-11e6-8f97-a4c19ebaba8c.png)

Notice that it has a `"scripts"` section but you do not.

**Comment by uniqueone2000** [comment]
I went to the documentation found here: 

Add the following package definition and configuration files to the project folder:

package.json lists packages the QuickStart app depends on and defines some useful scripts. See Npm Package Configuration for details.

which was right above the image you posted and copied it from there. I didn't even look though those tabs. When I did (just before replying to you here), I got it to work. Thanks so much for pointing out my mistake(s). It just goes to show that a little attention to detail goes a long way. Thank you so very much and my apologies for taking up your time. I will continue to follow along (hopefully with my eyes open this time, lol), with the rest of this tutorial and will at least give you an update on my progress.  THANKS AGAIN :-)
